content/download/lts.adoc

```diff
@@ -121,8 +121,8 @@ However, if you're using Jenkins 2.47 or Jenkins 2.56, you may have problems dow
 ### Switching Update Sites
 
 The Jenkins project operates multiple update sites that inform Jenkins about available updates to Jenkins and plugins.
-As Jenkins sends its current version when requesting new data, the same URL can be used for all releases of Jenkins, and will always serve the most appropriate update information.
+As Jenkins sends its current version when requesting new data, the same link:http://updates.jenkins-ci.org/update-center.json[URL] can be used for all releases of Jenkins, and will always serve the most appropriate update information.
 For example, instances running an LTS release of Jenkins will only be offered LTS versions of Jenkins to upgrade to.
 
-After switching between release lines, it's recommend to update the update site metadata cached in Jenkins.
+After switching between release lines, it's recommend to update the update site metadata cached in Jenkins by hitting the "Check Now" button in the Plugins Manager, or hitting this url: \http://<jenkins_base_URL>/pluginManager/checkUpdatesServer.
```

Comment:
I don't believe we advertise that URL anywhere; it's unlikely to change, but it's still an implementation detail.  That endpoint also requires a POST request, so it's not going to look too pretty in the browser.

So I'd probably stick to recommending that people use "Check now" (note that "now" isn't capitalised), which could also be a link to [`/doc/book/managing/plugins/#from-the-web-ui`](https://jenkins.io/doc/book/managing/plugins/#from-the-web-ui).